xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -114,11 +114,11 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The byte read-only span containing the compressed data.</param>
```

Comment:
Same comment on parameter type as noted above.

Reply:
If you agree with the one above, I hope you agree with this one too, since it's the same text 😄 :

```suggestion
        <param name="source">A buffer containing the compressed data.</param>
```